introduction.md

```diff
@@ -7,3 +7,7 @@ Angular
 Update more lines
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?